### PR TITLE
Allow to skip execution of FieldCoverageMonitor if no item is returned

### LINKED ISSUE #366: Allow to skip execution of FieldCoverageMonitor if no item is returned
This is a fork of issue #339 .

`FieldCoverageMonitor` requires that we have at least one item returned, or else it will fail. However there are situations when we accept that a spider return no items, but if items are returned, we want to ensure our field coverage. 

In a project that are in production, we have dozens of real estate websites searching for locations in different neighborhoods daily. Some neighborhoods sometimes doesn't return any item in a specific day, but when data is returned we want to ensure that all the fields has the right coverage.

One possible solution (that looks simpler to implement) would be add a new setting called `SPIDERMON_FIELD_COVERAGE_SKIP_IF_NO_ITEM` (or something similar). If set to True and no item is returned, field coverage will not be checked, otherwise, it will work as it is now.

### PR DESCRIPTION
**Pull request by VladAlex98**
Closes #366 

Added another setting called `SPIDERMON_FIELD_COVERAGE_SKIP_IF_NO_ITEM` for `FieldCoverageMonitor`:
- If this setting is enabled and we don't have any items scraped, it will not raise error anymore, it will send a message instead.

### CODE REVIEW

**Review by rennerocha — DISMISSED** [review]
Hello @VladAlex98 ,
Thanks for your contribution. I added a suggestion so the logs returned for the user would looks nicer and more consistent with other monitor results logs that we have in other built-in monitors.
Also, do not forget to add unit tests for your change and update the documentation of FieldCoverageMonitor adding this new setting.

**Comment by VladAlex98** [comment]
Hi @rennerocha, thanks for the code review. I updated the code, added docs and some tests. Please let me know if you need to add something else.

**Comment by VMRuiz** [comment]
Hello @rennerocha , is this good now?

File: spidermon/contrib/scrapy/monitors/monitors.py [hunk 1]
```diff
@@ -417,9 +420,17 @@ def run(self, result):
             raise NotConfigured(
                 "To enable field coverage monitor, set SPIDERMON_ADD_FIELD_COVERAGE=True in your project settings"
             )
+
         return super().run(result)
 
     def test_check_if_field_coverage_rules_are_met(self):
+        skip_no_items = self.crawler.settings.get(
```
Comment (curita) [inline spidermon/contrib/scrapy/monitors/monitors.py:427]: **suggestion:** We should use `.getbool()` here instead, so we can parse boolean-like strings as valid values too. That way, we'll support setting this value in ScrapyCloud or the command line.

**Review by curita — APPROVED** [review]
Changes look good!

I mentioned a small suggestion, but otherwise, the functionality makes sense to me.

**Pull request merged**